addons/mail/views/mail_activity_views.xml

```diff
@@ -12,7 +12,7 @@
                     </div>
                     <group>
                         <group>
-                            <field name="res_model_id"/>
+                            <field name="res_model_id" groups="base.group_no_one"/>
```

Comment:
Please move that in a separate commit. As we target a stable, each separate point should be in a separate commit.